js/components/dataset/filters.js

```diff
@@ -13,26 +13,11 @@ export default {
             }
             const start = moment(range.start);
             const end = range.end ? moment(range.end) : undefined;
-            const start_label = start.format('L');
-            const end_label = end.format('L');
+            const start_label = start.format('YYYY');
+            const end_label = end.format('YYYY');
 
-            // delta = start.diff(end)
-            // delta = value.end - value.start
-            // start, end = None, None
-            // if start.clone() is_first_year_day(value.start) and is_last_year_day(value.end):
-            //     start = value.start.year
-            //     if delta.days > 365:
-            //         end = value.end.year
-            // elif is_first_month_day(value.start) and is_last_month_day(value.end):
-            //     start = short_month(value.start)
-            //     if delta.days > 31:
-            //         end = short_month(value.end)
-            // else:
-            //     start = short_day(value.start)
-            //     if value.start != value.end:
-            //         end = short_day(value.end)
             return end_label
-                ? this._('{start} to {end}', {start:start_label, end:end_label})
+                ? this._('{start_label}-{end_label}', {start_label, end_label})
```

Comment:
This probably does not need to be translated anymore?

Reply:
Yes, correct!